_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+</aside>
+
+<pre><code class="o-syntax-highlight--scss">// Incomplete example for demo purposes only
+
+// Set the brand.
+$o-brand: 'whitelabel';
+
+// Customise: Tell the o-typography component to use our custom
+// font, it will affect all other components which
+// as they are built using o-typography.
+@include oTypographySetFont(
+    $type: 'sans',
+    $family: 'Comic Sans MS', sans-serif
+);
+
+// Customise: Tell the o-header-services we want hot pink
+// in the navigation.
+@include oHeaderServicesCustomize((
+	'nav-hover-background': hotpink
+));
+</code></pre>
+
+
+<figure>
+	<img alt="" src="/assets/images/component-customisation/whitelabel-customisation.png" />
+	<figcaption>
+        An example project which builds on the whitelabel brand. The font has been customised with `o-typography` Sass (oooh! Comic Sans). It affects all components which use `o-typography`, including the other components on the page, `o-table` and `o-header-services`. `o-header-services` has also been customised using Sass to have a unique highlight colour (oooh! hotpink!)
+	</figcaption>
+</figure>
+
+This is great to create a distinct brand, where a customisation should affect all instances of a component across a project.
+
+If your project is based on the master or internal brand, however, it is more likely that you want to create new variants of a component to use alongside existing ones. For example to output an additional banner style to support a new marketing campaign.
+
+## Themes/Variants
+
+If your project has access to a component's Sass (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) your project may use Sass mixins to customise components by outputting a new variant of that component. A component variant is an instance of a component with modified appearance or functionality.
+
+For instance, at the time of writing, the `o-banner` component has an [`oBannerAddTheme` mixin](https://registry.origami.ft.com/components/o-banner@3.4.2/sassdoc?brand=master#mixin-obanneraddtheme) which provides options for creating a banner theme with customised colour and background image:
+
+<pre><code class="o-syntax-highlight--scss">@include oBannerAddTheme('my-pikachu-theme', (
+    background-image: url('https://example.org/pikachu.png'),
+    background-color: oColorsByName('lemon'),
+    text-color: oColorsByName('slate'),
+    button-background-color: oColorsByName('crimson'),
+));</code></pre>
+
+<aside>If your project is using the <a href="/docs/tutorials/build-service/">Origami Build Service</a>, and does not have access to the component's Sass API, it is not currently possible to create custom component variants. It may be possible in the future using <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties">Custom CSS Properties (CSS Variables)</a>, when projects no longer support older browsers such as Internet Explorer.</aside>
```

Comment:
I read this aside as saying it is possible to use the Origami Build Service and access the Sass API, which is currently not true. We only offer the ability to set the brand and system_code Sass variables via the service.

```suggestion
<aside>If your project is using the <a href="/docs/tutorials/build-service/">Origami Build Service</a>, you do not have access to the component's Sass API, as such it is not currently possible to create custom component variants. It may be possible in the future using <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties">Custom CSS Properties (CSS Variables)</a>, when projects no longer support older browsers such as Internet Explorer.</aside>
```